svcdef/resolvetypes.go

```diff
@@ -0,0 +1,61 @@
+package svcdef
+
+// After the initial creation of Svcdef, each FieldType will have it's name
+// set to the name of the type that it represents, but it won't have a pointer
+// to the full instance of the type with that name. Type resolution is the
+// process of taking the name of each FieldType, then searching for the
+// enum/message with that name and setting the respective *Message/*Enum
+// property of our FieldType to point to that enum/message.
```

Comment:
This comment is currently just "floating" in the middle of nowhere, and isn't tied to this package or any functions within this file. This explanation should probably be moved into the doc comment for the `resolveTypes(*Svcdef)` function below.